README.md

```diff
@@ -46,6 +46,19 @@ gemrat --pessimistic cocoapods xcpretty gym deliver match
 
 ---
 
+## Upgrading CocoaPods in a legacy project
```

Comment:
I think this should go in the Troubleshooting section at the bottom. That's where I'd look for it.